parlai/utils/data.py

```diff
@@ -115,20 +115,15 @@ def is_streaming(cls, datatype: str) -> bool:
         return 'stream' in datatype
 
     @classmethod
-    def split_domains_by_fold(
+    def split_data_by_fold(
         cls,
         fold: str,
-        domains: List[List],
+        data: List,
         train_frac: float,
         valid_frac: float,
         test_frac: float,
         seed: int = 42,
     ):
-        """
-        Need to be careful about how we setup random to not leak examples between trains
```

Comment:
Please make a  docstring for this method.

Reply:
Done.